docs/new_assignment.rst

```diff
@@ -0,0 +1,57 @@
+Creating a new assignment template
+----------------------------------
+
+Creating a new assignment template involves getting your local assignment
+files into
+a new repository on GitHub that you can use as a template for a GitHub
```

Comment:
ok @kcranston 

1. does this assume you are already in the course directory where the config file is?
i went to a course directory but had not run quickstart there. i ran

`abc-assignment-template -a assignment1` 

and got
`abc-assignment-template: error: unrecognized arguments: -a`

Reply:
also what about calling this `abc-create-assignment-repo` or something a bit more expressive?

Reply:
also is -a for add or for name? 

in the code i see:

```
parser.add_argument(
        "assignment",
        help="Name of assignment. Must match name in nbgrader release directory",
    )
``` 

which suggests it is the name but a isn't as experssive so what about `-name assignment1`